js/Core/Options.js

```diff
@@ -993,29 +993,34 @@ H.defaultOptions = {
          * @sample  {highcharts} highcharts/chart/pankey/ Zooming and panning
          * @sample  {highstock} stock/chart/panning/ Zooming and xy panning
          *
-         * @product highcharts highstock gantt
+         * @product highcharts highstock highmaps gantt
```

Comment:
Listing all 4 products should be the same as not setting `@product` at all.
Example `chart.styledMode`:
 https://github.com/highcharts/highcharts/blob/a2d73c62eaef8f597f57e190d2e6801a2351eef4/js/Core/Options.js#L1041-L1051